catalog/pom.xml

```diff
@@ -2979,7 +2979,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.camel.quarkus</groupId>
-            <artifactId>camel-quarkus-qute</artifactId>
+            <artifactId>camel-quarkus-qute-component</artifactId>
```

Comment:
These virtual deps only serve as module ordering hints for mvnd or mvn -T. Moreover they are 100% generated. If you run `mvn process-resources -Pformat -N` from the repo root (which recommended after changing dependencies), the change will get reverted. Could you please explain your motivation for the change?

Reply:
Thanks Peter. I actually want to prevent the catalog from including a bogus `qute.json` that is causing this issue https://github.com/apache/camel-k-runtime/issues/677

Reply:
Let me try without this one.